src/script.h

```diff
@@ -77,10 +77,11 @@ struct scriptRunCtx {
 };
 
 /* Scripts flags */
-#define SCRIPT_FLAG_NO_WRITES (1ULL<<0)
-#define SCRIPT_FLAG_ALLOW_OOM (1ULL<<1)
-#define SCRIPT_FLAG_ALLOW_STALE (1ULL<<3)
-#define SCRIPT_FLAG_NO_CLUSTER (1ULL<<4)
+#define SCRIPT_FLAG_IGNORE_FLAGS (1ULL<<0) /* EVAL Script backwards compatible behavior, no shebang provided */
```

Comment:
I don't like the name of this flag.
it's not that we ignore flags, it's that we run in some "compatibility" mode.
how about SCRIPT_FLAG_EVAL_COMPAT_MODE?
also, i don't see any special reason why this must be the first flag, let's put it last (and change less lines).

Reply:
ok, makes sense. I names it that way when I required `flags=` in the shebang in order to be in the new more. So it really was "ignore flags" but not that any shebang moves us out of compatibility mode then the new name makes sense. I'll fix.

Reply:
I'm still going to change all these lines because I don't like the alignment but even worse I see no reason for the original numbers to skip (1ULL<<2) (see the numbers go 0,1,3,4 and skip 2? weird).